### ISSUE TITLE
Possible confusion of *x* and *y* coordinates w.r.t. to 3D image matrix

### ISSUE THREAD

**Issue by pinakinathc** [tzutalin/labelImg#333]
- **OS:Ubuntu 16.04**
- **Python3**
- **PyQt version:**
in labelImg, if we move the mouse pointer towards right, the *x* coordinate increases and if we move the mouse pointer down the *y* coordinate increases.

Now if we do the following:
`import cv2`
`img = cv2.imread(filename)`
and
`x = img.shape[0]`
`y = img.shape[1]`

here you see, if you go right, then according to `cv2` you *y* increases but according to labelImg *x* increases.

Basically there is a possible confusion in the *rows* and *columns*

**Comment by vdalv** [comment]
The confusion is that you're under the impression that:
```
x = img.shape[0]
y = img.shape[1]
```

When in fact the opposite is true:
```
y = img.shape[0]
x = img.shape[1]
```

Sources:
https://docs.opencv.org/3.0-beta/doc/py_tutorials/py_core/py_basic_ops/py_basic_ops.html
https://www.tutorialkart.com/opencv/python/opencv-python-get-image-size/

**Comment by pinakinathc** [comment]
@vdalv you mean that you refer a 2D matrix as:
`img[y][x]` to spot a pixel

but probably might be a source of confusion as I am used to access an element in 2D matrix as:
`img[x][y]`

Came across this confusion while reading the code in another git repo where they used:
`img[x][y]` format instead of `img[y][x]`

Reason for posting it: It would probably be a good idea to mention this format in docs to evade the possible source of confusion.

**Comment by vdalv** [comment]
@pinakinathc My comment was in regards to your OpenCV example, and this statement: "here you see, if you go right, then according to cv2 you y increases but according to labelImg x increases."

In any case, there is no confusion/bug in the code.
